Assignment3/A3ClassLibrary/A3ClassLibrary/JsonLoc.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
```

Comment:
Suggest modularizing the program into classes and letting every each class serve only one purpose.